packages/iov-ethereum/webpack.web.config.js

```diff
@@ -0,0 +1,21 @@
+const glob = require('glob');
+const path = require('path');
+const webpack = require('webpack');
+
+const target = "web";
+const distdir = path.join(__dirname, "dist", "web");
+
+module.exports = [
+  {
+    // bundle used for Karma tests
+    target: target,
+    entry: glob.sync("./build/**/*.spec.js"),
+    output: {
+      path: distdir,
+      filename: "tests.js",
+    },
+    plugins: [
+      new webpack.EnvironmentPlugin(['LONG_RUNNING_ENABLED']),
```

Comment:
The environment variable `LONG_RUNNING_ENABLED` is not used in the new tests, so we don't need to pass it from environment to Karma tests. But what we probably want to have is `ETH_ENV`